acceptance/acceptance_test.go

```diff
@@ -15,13 +17,83 @@ import (
 	h "github.com/buildpacks/lifecycle/testhelpers"
 )
 
+type lifecycleCmd func(binary string, args ...string) *exec.Cmd
+
 func TestAcceptance(t *testing.T) {
-	spec.Run(t, "acceptance", testAcceptance, spec.Parallel(), spec.Report(report.Terminal{}))
+	buildDir, err := ioutil.TempDir("", "lifecycle-acceptance")
+	h.AssertNil(t, err)
+	defer h.AssertNil(t, os.RemoveAll(buildDir))
+	buildBinaries(t, buildDir)
+
+	spec.Run(t, "acceptance", func(t *testing.T, when spec.G, it spec.S) {
+		testAcceptance(t, when, it, func(binary string, args ...string) *exec.Cmd {
+			return exec.Command(filepath.Join(buildDir, "lifecycle", binary), args...)
+		})
+	}, spec.Parallel(), spec.Report(report.Terminal{}))
 }
 
-func testAcceptance(t *testing.T, when spec.G, it spec.S) {
+func testAcceptance(t *testing.T, when spec.G, it spec.S, lifecycleCmd lifecycleCmd) {
+	var tmpDir string
+
+	it.Before(func() {
+		var err error
+		tmpDir, err = ioutil.TempDir("", "acceptance-*")
+		h.AssertNil(t, err)
+	})
+
+	it.After(func() {
+		os.RemoveAll(tmpDir)
+	})
 
 	when("All", func() {
+		when("CNB_PLATFORM_API", func() {
+			tmpWorkingDir := func(sourceDir string) string {
+				d, err := ioutil.TempDir(tmpDir, "")
+				h.AssertNil(t, err)
+				h.RecursiveCopy(t, sourceDir, d)
```

Comment:
If the only purpose of this is to provide a `group.toml` file, why not just pass `-group=<path/to/test/fixture>` instead of moving this to the default path